libs/dh/admin/feature-user-roles-new/src/index.ts

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 export * from './lib/dh-user-roles.component';
-export { DhCreateUserRoleModalComponent } from './lib/create/dh-create-user-role-modal.component';
+export { DhCreateUserRoleComponent as DhCreateUserRoleModalComponent } from './lib/create.component';
```

Comment:
suggestion: `DhCreateUserRoleModalComponent` will need to be replaced with `DhCreateUserRoleComponent` wherever it's being used.

```suggestion
export { DhCreateUserRoleComponent } from './lib/create.component';
```